spec/components/schools/teacher_profile_summary_list_component_spec.rb

```diff
@@ -1,12 +1,22 @@
 require "rails_helper"
 
 RSpec.describe Schools::TeacherProfileSummaryListComponent, type: :component do
-  include TeacherHelper
-  include ECTHelper
-
-  let(:ect) { FactoryBot.create(:ect_at_school_period, :active, started_on: 3.years.ago) }
+  let(:teacher) { FactoryBot.create(:teacher, trn: '9876543', trs_first_name: 'Kakarot', trs_last_name: 'SSJ') }
+  let(:teacher_2) { FactoryBot.create(:teacher, trn: '987654', trs_first_name: 'Naruto', trs_last_name: 'Ninetails') }
   let(:mentor) { FactoryBot.create(:mentor_at_school_period, :active, started_on: 3.years.ago) }
-  let!(:ongoing_mentorship) { FactoryBot.create(:mentorship_period, :active, mentee: ect, mentor:) }
+  let(:mentor_2) { FactoryBot.create(:mentor_at_school_period, :active, teacher: teacher_2, started_on: 3.years.ago) }
+  let(:ect) do
+    FactoryBot.create(:ect_at_school_period, :active, teacher:, started_on: Date.new(2021, 9, 1),
+                                                      email: 'foobarect@madeup.com', working_pattern: 'full_time')
+  end
+
+  let!(:ongoing_mentorship) do
```

Comment:
Is doing this any clearer?

`teacher` > `mentee_teacher`
`teacher_2` > `mentor_teacher`
`mentor` > `previous_mentor`
`mentor_2` > `current_mentor` 
 `ect` > `mentee`

And just building the `mentorship_periods` in a `before` block because we don't access them by variable name.

Reply:
Yes, definitely! Thank you 🙏🏽 

You can clearly tell I do not know the domain 😅 

Updated here: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/313/commits/9663112674b1614e637342ac73d3a29a16168044